src/libs/ReportScrollManager/index.js

```diff
@@ -15,8 +16,15 @@ function scrollToIndex(index, isEditing) {
     if (isEditing) {
         return;
     }
-
-    flatListRef.current.scrollToIndex(index);
+    const sizeMap = flatListRef.current.props.sizeMap;
+    const clientHeight = flatListRef.current.getScrollableNode().clientHeight;
+    const paddingTop = parseFloat(flatListRef.current.getScrollableNode().childNodes[0].style.paddingTop || '0px');
```

Comment:
We should access nested properties using `lodashGet`
More details in [STYLING.md](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#accessing-object-properties-and-default-values)